packages/analytics-browser/src/attribution/campaign-tracker.ts

```diff
@@ -80,33 +88,38 @@ export class CampaignTracker implements ICampaignTracker {
       return identify.unset(key);
     }, new Identify());
 
-    const pageViewEvent: BaseEvent = {
-      event_type: 'Page View',
-      event_properties: {
-        page_title: /* istanbul ignore next */ (typeof document !== 'undefined' && document.title) || '',
-        page_location: /* istanbul ignore next */ (typeof location !== 'undefined' && location.href) || '',
-        page_path: /* istanbul ignore next */ (typeof location !== 'undefined' && location.pathname) || '',
-      },
-    };
-    return {
-      ...createIdentifyEvent(identifyEvent),
-      ...(this.trackPageViews && pageViewEvent),
-    };
+    return createIdentifyEvent(identifyEvent);
   }
 
-  async send(isNewSession: boolean) {
-    if (this.disabled) {
-      return;
-    }
-    const currentCampaign = await this.parser.parse();
-    const previousCampaign = await this.getCampaignFromStorage();
-    if (!isNewSession) {
-      if (!this.trackNewCampaigns || !this.isNewCampaign(currentCampaign, previousCampaign)) {
-        return;
+  trackOn(filter: PageTrackingFilter = undefined, callback: (currentCampaing: Campaign) => unknown) {
```

Comment:
typo - currentCampaing

Reply:
`trackOn` feels confusing. I think the places that call it currently would be simplified if they just called getCurrentState directly and did like `if (isNewCampaign){...`. The attribution plugin doesn't need to interact with code related to the pageview plugin that way.